.travis.yml

```diff
@@ -11,26 +11,26 @@ jobs:
     - name: Run plugin test suite with python 3.7
       python: 3.7
       script: 'pytest'
-
-    - name: Typecheck Django 3.0 test suite with python 3.8
-      python: 3.8
-      script: |
-        python ./scripts/typecheck_tests.py --django_version=3.0
-
-    - name: Typecheck Django 3.0 test suite with python 3.7
-      python: 3.7
-      script: |
-        python ./scripts/typecheck_tests.py --django_version=3.0
-
-    - name: Typecheck Django 3.0 test suite with python 3.6
-      python: 3.6
-      script: |
-        python ./scripts/typecheck_tests.py --django_version=3.0
-
-    - name: Typecheck Django 2.2 test suite with python 3.7
-      python: 3.7
-      script: |
-        python ./scripts/typecheck_tests.py --django_version=2.2
+#
```

Comment:
Were these tests commented out by accident? @mkurnikov

Reply:
I mentioned it in the proposal, testing stubs against test suite won't work anymore.  
That's why before merging this, we have to some kind of merging tool. 

I look into it, maybe I'm wrong, and everything's okay.